.github/workflows/ci.yml

```diff
@@ -34,9 +34,9 @@ jobs:
 
   test:
     strategy:
-      matrix: 
+      matrix:
         os: [ 'macos-latest', 'ubuntu-latest', 'windows-latest' ]
-        go: [ '1.17' ]
+        go: [ '1.18', '1.19' ]
```

Comment:
Is there a reason you wanted to test both versions rather than just 1.19?

Reply:
The idea is to test the earliest supported and the latest.

Reply:
If you only want to support the latest, I can remove 1.18.

Reply:
Oh ok I see, that's makes sense. And given I'm still on 1.18 on my home PC I'd actually prefer we keep both.